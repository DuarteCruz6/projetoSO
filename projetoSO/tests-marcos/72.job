BACKUP
WRITE [(w4,69)(k1,2)(y6,65)(n6,44)]
READ [n6]
READ [n6]
DELETE [w4,k1]
WRITE [(y2,65)(n7,91)(z3,76)(c9,26)(u3,65)]
BACKUP
WRITE [(t0,9)]
READ [n6]
WRITE [(u5,24)(s7,56)]
WRITE [(y9,41)(w7,78)(f0,39)]
READ [c9]
WRITE [(a4,89)(m8,47)(x8,20)(e7,90)(u1,94)]
BACKUP
WRITE [(v2,7)(v8,17)(w4,74)(i4,96)]
WRITE [(z5,3)(o5,7)(p6,91)]
WRITE [(f4,92)(e5,18)(j3,44)(i1,46)]
WRITE [(l5,16)(n9,19)]
READ [n9]
WRITE [(j2,49)(o3,6)(k2,0)(y3,81)(q3,69)]
WRITE [(w6,63)(w3,90)(d8,41)(x9,86)]
WRITE [(b8,90)]
READ [y2]
READ [l5]
READ [f0]
SHOW
WRITE [(i6,92)(h1,84)]
WRITE [(r5,71)(l1,94)]
WRITE [(p4,86)(q9,38)]
