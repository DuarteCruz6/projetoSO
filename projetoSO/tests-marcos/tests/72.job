WRITE [(g4,92)(o5,74)]
READ [g4]
WRITE [(o8,59)(a2,12)(m9,52)]
DELETE [m9]
SHOW
WRITE [(r2,9)]
READ [a2]
WRITE [(a5,59)(t2,39)(v0,79)]
READ [o5]
WRITE [(w4,96)(o9,64)]
DELETE [o5,r2,t2]
DELETE [v0,o8,a5,a2,w4]
READ [o9]
WRITE [(k7,91)(s5,71)(t6,42)(g8,81)(j5,19)]
DELETE [o9,j5,s5,g4]
SHOW
WRITE [(p2,10)(l7,3)(m2,91)(i6,34)]
DELETE [k7,l7,t6]
WRITE [(s6,39)(m7,13)]
WRITE [(b5,18)(l4,84)(l2,70)]
WRITE [(d5,87)(n9,95)(o6,82)(l1,67)]
READ [i6]
READ [m2]
DELETE [l1]
WRITE [(b7,10)(n6,27)]
BACKUP
WRITE [(f7,66)(b2,88)(d4,83)(x1,50)]
WRITE [(f5,94)]
WRITE [(v9,67)]
WAIT 2000
WRITE [(s9,98)(z6,64)]
WRITE [(c8,20)(e3,35)]
READ [b5]
WRITE [(i8,65)(t0,40)]
WRITE [(k5,57)(n8,76)]
WRITE [(w2,23)(v4,37)(j3,80)]
WRITE [(f2,12)]
DELETE [s9]
WRITE [(z5,57)(f0,12)]
DELETE [z6,f7]
WRITE [(z8,53)(e4,18)(i2,93)(t3,90)]
WRITE [(p1,39)(e9,97)]
DELETE [e9,j3,n9,b7]
WRITE [(p9,66)]
WRITE [(a0,86)(f7,21)(m1,47)(v1,67)(a8,27)]
READ [p1]
SHOW
