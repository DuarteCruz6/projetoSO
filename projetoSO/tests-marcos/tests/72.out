[(g4,92)]
(a2, 12)
(g4, 92)
(o8, 59)
(o5, 74)
[(a2,12)]
[(o5,74)]
[(o9,64)]
(g8, 81)
(k7, 91)
(t6, 42)
[(i6,34)]
[(m2,91)]
[(b5,18)]
[(p1,39)]
(a8, 27)
(a0, 86)
(b2, 88)
(b5, 18)
(c8, 20)
(d4, 83)
(d5, 87)
(e4, 18)
(e3, 35)
(f7, 21)
(f0, 12)
(f2, 12)
(f5, 94)
(g8, 81)
(i2, 93)
(i8, 65)
(i6, 34)
(k5, 57)
(l2, 70)
(l4, 84)
(m1, 47)
(m7, 13)
(m2, 91)
(n8, 76)
(n6, 27)
(o6, 82)
(p9, 66)
(p1, 39)
(p2, 10)
(s6, 39)
(t3, 90)
(t0, 40)
(v1, 67)
(v4, 37)
(v9, 67)
(w2, 23)
(x1, 50)
(z8, 53)
(z5, 57)
